app/controllers/candidate_interface/find_course_selections_controller.rb

```diff
@@ -44,11 +44,11 @@ def pick_site_for_course(course, course_option_id)
         course_option_id: course_option_id,
       )
 
-      if pick_site.save
-        redirect_to candidate_interface_course_choices_index_path
-      else
-        render :options_for_site
+      unless pick_site.save
+        flash[:warning] = "You cannot have more than 3 course choices. You must delete a choice if you want to apply to #{course.name_and_code}."
```

Comment:
Should we use an error from the `PickSiteForm` itself? Less responsibility for the controller.

The string's also in `CandidateInterface::SignInController` — i18n?

Reply:
Yes, I was being lazy 👍